awswrangler/timestream.py

```diff
@@ -148,8 +161,9 @@ def write(
     database: str,
     table: str,
     time_col: str,
-    measure_col: str,
-    dimensions_cols: List[str],
+    measure_col: Optional[str] = None,
+    measure_cols: Optional[List[str]] = None,
+    dimensions_cols: Optional[List[str]] = None,
```

Comment:
Any reason why this had to become Optional? Especially that an exception is raised on L230 if it's not provided...

Reply:
Yes, the reason is that you can't follow mandatory args with optional. The alternative is to change the order which is way worse.

Reply:
Yeah, good idea